layouts/partials/features.html

```diff
@@ -10,11 +10,21 @@
 		{{ end }}
 				<div class="col-md-4">
                     <div class="box-simple">
+                      {{ if .url }}
```

Comment:
@jcolomb thanks for the PR.

I don't like repeat code, what do you think in append always the block with something like:
```
<a href="{{ .url | default: "" }}">
   BLOCK
</a>
```

Reply:
I like it.

Does Hugo have a `for..each..in` equivalent btw?

Cause that way we could overhaul it to create a object `['link1.com','example.org','google.com']` from the `FILENAME.yaml`.

We could even potentially modify the whole https://github.com/devcows/hugo-universal-theme/tree/master/exampleSite/data/features section to run from just one features.yaml file if that was a thing, so it would just have a list of sections in one file with 
```
weight: 4
 name: "Consulting"
 icon: "fa fa-lightbulb-o"
+url: "/consulting/"
 description: "Fifth abundantly made Give sixth hath. Cattle creature i be don't them behold green moved fowl Moved life us beast good yielding. Have bring."
```
on each section.

https://gohugo.io/templates/introduction/#iteration